batch/batch/driver/create_instance.py

```diff
@@ -109,7 +125,7 @@ async def create_instance(
                     'key': 'run_script',
                     'value': rf'''
 #!/bin/bash
-set -x
+set -ex
```

Comment:
I'm not sure if we need this change.

Reply:
I think this is good we should fail early.